### PR TITLE
added environment variables

### PR DESCRIPTION
**Pull request by GanadiniAkshay**
Added documentation for using environment variables

### CODE REVIEW

File: docs/bot-builder/integrations/integrating-custom-code.md [hunk 1]
```diff
@@ -128,7 +128,18 @@ When you click on the `Run` button, the code will be executed using the sample i
...
+### Using Environment Variables
+It is also possible to set environment variables for the code executor to store values like database username and passwords that you don't want to expose inside the code.
 
+**The environment variables are stored at a bot level.** This means that varibales stored in one node are automatically available to all nodes inside the bot.
+
+You can set these variables by opening the code editor, clickin on the `More` button in the top right corner and selecting the `Variables` option.
+
+![Environment Variables in Code Editor](assets/env_variables_create.gif)
+
+In the variables menu that opens, you can set the variable name and values for the vraible on the staging and production environments. For example you can create a variable called `MONGO_URI` and set its staging value as `mongodb://mongodb0.example.com:27017/admin` and it's production value as `mongodb://mongodb1.example.com:27017/admin`.
```
Comment (sandesh89) [inline docs/bot-builder/integrations/integrating-custom-code.md]: Minor typo in "vraible"

File: docs/bot-builder/integrations/integrating-custom-code.md [hunk 2]
```diff
@@ -128,7 +128,18 @@ When you click on the `Run` button, the code will be executed using the sample i
...
 ![Code Run](assets/integrating_custom_code_run.gif)
 
+### Using Environment Variables
+It is also possible to set environment variables for the code executor to store values like database username and passwords that you don't want to expose inside the code.
 
+**The environment variables are stored at a bot level.** This means that varibales stored in one node are automatically available to all nodes inside the bot.
+
+You can set these variables by opening the code editor, clickin on the `More` button in the top right corner and selecting the `Variables` option.
+
+![Environment Variables in Code Editor](assets/env_variables_create.gif)
```
Comment (sandesh89) [inline docs/bot-builder/integrations/integrating-custom-code.md]: Do we generally specify what permission this has ? or feature name

File: docs/bot-builder/integrations/integrating-custom-code.md [hunk 3]
```diff
@@ -128,7 +128,18 @@ When you click on the `Run` button, the code will be executed using the sample i
 
 ![Code Run](assets/integrating_custom_code_run.gif)
 
+### Using Environment Variables
+It is also possible to set environment variables for the code executor to store values like database username and passwords that you don't want to expose inside the code.
 
+**The environment variables are stored at a bot level.** This means that varibales stored in one node are automatically available to all nodes inside the bot.
+
+You can set these variables by opening the code editor, clickin on the `More` button in the top right corner and selecting the `Variables` option.
```
Comment (sandesh89) [inline docs/bot-builder/integrations/integrating-custom-code.md]: Typo: clickin

File: docs/bot-builder/integrations/integrating-custom-code.md [hunk 4]
```diff
@@ -128,7 +128,18 @@ When you click on the `Run` button, the code will be executed using the sample i
 
 ![Code Run](assets/integrating_custom_code_run.gif)
 
+### Using Environment Variables
+It is also possible to set environment variables for the code executor to store values like database username and passwords that you don't want to expose inside the code.
 
+**The environment variables are stored at a bot level.** This means that varibales stored in one node are automatically available to all nodes inside the bot.
```
Comment (sandesh89) [inline docs/bot-builder/integrations/integrating-custom-code.md]: Typo on : varibales

File: docs/bot-builder/integrations/integrating-custom-code.md (hunk 2 above)
Comment (GanadiniAkshay) [reply docs/bot-builder/integrations/integrating-custom-code.md]: No that is hidden. And right now all bot builders have access to this so not sure what specific value we can add

**Pull request merged**